Chunking.md

```diff
@@ -0,0 +1,84 @@
+# SaltyRTC Chunking
+
+This specification describes the binary data chunking algorithm used by
+the SaltyRTC WebRTC task. It allows the user to split up large binary
+messages into multiple chunks of a certain size.
+
+## Conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
+document are to be interpreted as described in
+[RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## Chunk Size
+
+The term *chunk size* is referring to the number of bytes of the *chunk
+header* and the *chunk data* combined.
+
+## Chunk Header
+
+When converting data to chunks, a 9 byte header MUST be prepended to
+each chunk. This allows for sending chunk over a network in any order.
+
+### Length
+
+The header is 9 bytes long.
+
+### Format
+
+The header is encoded in binary using network-oriented format (most
+significant byte first, also known as big-endian). It is structured as
+follows:
+
+    |O|IIII|SSSS|
+
+    - O: Options bit field (1 byte)
+    - I: Message id (4 bytes)
+    - S: Serial number (4 bytes)
```

Comment:
To rule out confusion, I'd say

> Message id (32 bit unsigned integer)
> Serial number (32 bit unsigned integer)

or alternatively add *32 bit unsigned integer* to the corresponding field description.